mixers/BaseMixer.sc

```diff
@@ -152,4 +152,18 @@ BaseMixer : Object {
             }).play(server, [\buffer, b]);
         }, oscPath);
     }
+
+    // add OSC listeners for each synth control available
+    addOSCControls { | oscpath, synthName, node |
```

Comment:
I ended up not using this, but keeping the code in there for reference and possible future use